Genomic surveillance for *Vibrio cholerae* using Vibriowatch
============================================================

**Vibriowatch** is a database of high quality genomes and metadata for *Vibrio cholerae*, with robust bioinformatics capability for virulence, antimicrobial resistance, and finding related isolates.

Vibriowatch contains `>4000` published *V. cholerae* genomes, and manually-curated metadata from the literature, including strain names; place, date and source of isolation; and phenotypic data for biotype, serogroup, serotype and antimicrobial resistance.

Vibriowatch is aimed for use by clinicians and public health workers studying cholera outbreaks, as well as for research biologists investigating *V. cholerae* genomic variation and evolution.

Vibriowatch's home is the `PathogenWatch`_ website.

.. _PathogenWatch: https://pathogen.watch/

In this tutorial, we will describe how users can use Vibriowatch to:

* upload reads or assembled genomes.
* verifying that their isolate's species is *V. cholerae*.
* view the statistics for their assembly.
* carry out MLST analysis of their isolate.
* compare their isolate to Vibriowatch's *V. cholerae* reference genomes.
* build a phylogenetic tree to check if their isolate belongs to the pandemic lineage.
* use cgMLST to find closely related isolates.
* build a phylogenetic tree of their isolate and its closest relatives.
* predict virulence genes.
* predict antimicrobial resistance genes.
* predict plasmids.

Check out the :doc:`usage` section for further information, including
how to :ref:`installation` the project.

.. note::

   This project is under active development.

Contents
--------

.. toctree::

   usage
   api
