Cholera, *Vibrio cholerae*, and *V. cholerae* Lineages
======================================================

Introduction
------------

This chapter will give you an overview of cholera, the species *Vibrio cholerae*, the main lineages of *V. cholerae*, and the current pandemic lineage of *V. cholerae*, 7PET.

Acute Watery Diarrhoea, the Disease Cholera and its Epidemiology
----------------------------------------------------------------

Cholera is a disease characterised by acute watery diarrhoea. It is an infection caused by a water-borne bacterium
called *Vibrio cholerae*, a curved Gram-negative rod-shaped bacterium.

.. figure:: Vibrio_cholerae.jpg
  :width: 350

  The bacterium *V. cholerae*. Image credit: Tom Kirn, Ron Taylor, Louisa Howard - Dartmouth Electron Microscope Facility.

The *V. cholerae* bacterium is usually transmitted by the faecal-oral route, that is, 
people usually catch cholera by eating food or by drinking water that has been contaminated
with human faeces containing *V. cholerae*.

.. figure:: FaecalOralRoute.jpg
  :width: 500

  The faecal-oral route for disease transmission. The orange lines represent barriers, ie. toilets, safe water, hygiene and handwashing. Image credit: Howard G., Bartam J., Williams A., Overbo A., Fuente D., Geere J.A. Domestic water quantity, service level and health, second edition. Geneva: World Health Organization; 2020.

Cholera outbreaks typically occur in places where there is poor access
to clean water and poor sewage infrastructure. These include low-income settings in which there is poor water sanitation infrastructure, 
or places in which there is a humanitarian crisis (e.g. an earthquake or war) that has disrupted the water sanitation infrastructure.

.. figure:: Wastewater.jpg
  :width: 350

  Children playing in a waste water drainage area that was found to be contaminated with *V. cholerae*. Image credit: Amibola Aman-Oloniyo, Nigeria.

Once ingested by a human host, the *V. cholerae* bacteria multiply in the human intestine. The
bacteria attach to the epithelial cells of the intestine and release **cholera toxin**. Cholera toxin
binds to the intestinal epithelial cell surface, and stimulates the cells to secrete ions and water into
the intestinal lumen, resulting in acute watery diarrhoea. Cholera toxin is therefore the key **virulence factor** 
of *V. cholerae*. 

.. figure:: CholeraToxin.jpg
  :width: 450

  Cholera toxin (Ctx) is a protein complex that consists of one CtxA protein bound to five CtxB proteins. This image shows the structure formed by the five CtxB proteins. Image credit: Wikipedia user Astrojan.

Note that *V. cholerae* is not the only pathogen that can cause acute watery diarrhoea; similar symptoms also be caused by other bacteria
such as enterotoxigenic *E. coli* (ETEC). 

The Species *Vibrio cholerae* and its Geographic Distribution
-------------------------------------------------------------

The species *V. cholerae* is distributed globally, and is a very diverse species with many different lineages (i.e. many different
genetically different subtypes of *V. cholerae*, forming different branches of the family tree of the species *V. cholerae*). 

There is there is only one lineage of *V. cholerae* which causes pandemic/epidemic cholera at present.
This lineage is known as the **'7PET'** lineage. 
The 7PET lineage is an extremely infectious and virulent lineage of *V. cholerae*, which produces
cholera toxin. Since it arose in the 1960s, 7PET appears to have evolved to become a 
human pathogen (`Feng et al 2008`_, `Chun et al 2009`_, `Hu et al 2016`_, `Mutreja & Dougan 2020`_).

There are many other non-pandemic/non-epidemic lineages of *V. cholerae* found around the world that do not cause pandemic/epidemic
cholera. The vast majority of these non-pandemic/non-epidemic lineages do not produce cholera toxin.

.. figure:: GlobalDistribution.jpg
  :width: 650

  The global distribution of the bacterium *V. cholerae*, including representatives of many different lineages of *V. cholerae*, including the current pandemic lineage (7PET) and many other lineages. Triangles indicate where *V. cholerae* was detected. The studies shown here are actually only a small fraction of the studies published for certain areas. Image credit: `Lutz et al 2013`_.

These non-pandemic/non-epidemic lineages of *V. cholerae* are often found in brackish water or in saltwater in association with shellfish such as crabs and shrimp, or in rivers or freshwater lakes. 

.. _Chun et al 2009: https://pubmed.ncbi.nlm.nih.gov/19720995/

.. _Feng et al 2008: https://pubmed.ncbi.nlm.nih.gov/19115014/

.. _Hu et al 2016: https://pubmed.ncbi.nlm.nih.gov/27849586/

.. _Mutreja & Dougan 2020: https://pubmed.ncbi.nlm.nih.gov/31345641/

.. figure:: VibrioCholeraeInSea.jpg
  :width: 750

  Non-pandemic/non-epidemic lineages of *V. cholerae* are part of the bacterioplankton in aquatic environments. Image credit: `Lutz et al 2013`_.

.. _Lutz et al 2013: https://pubmed.ncbi.nlm.nih.gov/24379807/

The History of Cholera: the Seven Historically Recorded Pandemics
-----------------------------------------------------------------

A disease with symptoms like cholera was recorded in ancient texts from ancient India and China, but we have
little information about those ancient outbreaks.
Since the early 1800s there have been seven historically recorded cholera pandemics (epidemics covering a very wide
area of our planet). The 7th pandemic is the current
and ongoing cholera pandemic, and has been caused by the 7PET lineage.

The 6th pandemic occurred from 1899 to 1923, after which there were only localised outbreaks in parts of Asia for several decades,
and then the 7th pandemic started in 1961 and continues at present (`Feng et al 2008`_). 
We now know that the 6th pandemic  
was actually caused by completely a different **lineage** of *V. cholerae* than the 7th pandemic. 
We call the lineage that caused the 6th pandemic the **'Classical lineage'** (`Chun et al 2009`_, `Domman et al 2017`_). 

.. _Domman et al 2017: https://pubmed.ncbi.nlm.nih.gov/29123068/

.. _Chun et al 2009: https://pubmed.ncbi.nlm.nih.gov/19720995/

The Classical lineage has more or less died out since the end of the 6th pandemic (in 1923) 
and is very rarely seen nowadays (e.g. a rare Classical lineage isolate 
from Ghana in 2010 was reported by `Adade et al 2022`_). 

.. _Dorman 2020: https://www.sanger.ac.uk/theses/md25-thesis.pdf

.. _Adade et al 2022: https://pubmed.ncbi.nlm.nih.gov/36312941/

.. _Feng et al 2008: https://pubmed.ncbi.nlm.nih.gov/19115014/  

.. figure:: SevenPandemics.png
  :width: 650

  A timeline of the seven historically recorded pandemics. The current pandemic is the 7th pandemic and is caused by the 7PET lineage. Image credit: A. Coghlan.

.. figure:: SnowCholeraMap.jpg
  :width: 500
  
  Map of cholera cases in London in 1854 made by the doctor John Snow during the 3rd historically recorded cholera pandemic. Cholera cases are highlighted in black. Image credit: Original map made by John Snow in 1854.

Diarrhoeal illness caused by different lineages of *V. cholerae*
----------------------------------------------------------------

The 7PET lineage, which has caused the current pandemic, and the Classical lineage, which caused the 6th historically
recorded pandemic, are only the two pandemic/epidemic-causing lineages of *V. cholerae*. The many other
lineages of *V. cholerae* are not pandemic/epidemic-causing; they sometimes cause relatively small outbreaks of diarrhoeal illness in
tens or even a hundred or so people (`Glenn Morris 1990`_). In contrast, 7PET is the only current *V. cholerae* lineage
that cuases large epidemics or pandemics of hundreds of thousands of cases, or even millions of 
cases as seen in the Yemen cholera epidemic from 2016-present (`Mutreja & Dougan 2020`_, `Lassalle et al 2023`_). 

.. _Mutreja & Dougan 2020: https://pubmed.ncbi.nlm.nih.gov/31345641/

.. _Glenn Morris 1990: https://pubmed.ncbi.nlm.nih.gov/2286218/

.. _Lassalle et al 2023: https://pubmed.ncbi.nlm.nih.gov/37770747/

Of the non-pandemic/non-epidemic *V. cholerae* lineages, the two lineages that have caused the most cases of diarrhoeal illness
in the last decade are thought to lineages 'L3b' and 'L9' (`Hao et al 2023`_). For example, these two non-pandemic/non-epidemic lineages
have caused several hundred cases of relatively mild diarrhoeal illness in Hangzhou, China between 2001 and 2018 (`Hao et al 2023`_).
Lineage L3b has also been linked to relatively small outbreaks of diarrhoeal illness in South Africa (`Smith et al 2021`_). 

.. _Hao et al 2023: https://pubmed.ncbi.nlm.nih.gov/37146742/

.. _Smith et al 2021: https://pubmed.ncbi.nlm.nih.gov/34670657/

.. figure:: L3b_Hangzhou.png
  :width: 600

  Cases of diarrhoeal illness per year caused by the L3b and L9 lineages of *V. cholerae* in Hangzhou, China, between 2001 and 2018. The grey lines represent the total number of diarrhoeal cases caused by L3b and L9 together, the blue lines represent the number of cases caused by L3b, and the orange lines the number of cases caused by L9. Image credit: `Hao et al 2023`_.

However, compared cholera outbreaks/epidemics caused by 7PET, outbreaks of L3b/L9 and other non-pandemic/non-epidemic
*V. cholerae* lineages are far smaller and in general involve milder symptoms and
rarely cause deaths. For example, the cholera epidemic in Yemen that began in 2016 (and is still continuing) has included
approximately million suspected cholera cases and appproximately 4000 deaths from 2016-2020 (`WHO 2020`_).

.. _WHO 2020: https://applications.emro.who.int/docs/WHOEMCSR314E-eng.pdf

.. figure:: YemenCholera.png
  :width: 600

  Number of suspected cases of cholera per week, during the cholera epidemic caused by 7PET in Yemen, between 2016 and 2020. Image credit: (`WHO 2020`_). 

.. _WHO 2020: https://applications.emro.who.int/docs/WHOEMCSR314E-eng.pdf

Due to its virulence and pandemic/epidemic-causing potential, the 7PET lineage is of major public health concern,
while the other non-pandemic/epidemic lineages of *V. cholerae* are in comparison only of relatively minor public health concern.

Relationships between lineages of *V. cholerae*
-----------------------------------------------

The figure below represents the 'family tree' of some of the known lineages of *V. cholerae*. It is actually a 'phylogenetic tree', that
is, a family tree based on the genetic information (DNA data) for representatives of the lineages. 

.. figure:: Lineages.png
  :width: 250
  
  A phylogenetic tree of some of the known lineages of *V. cholerae*.

-clear explanation of tree for the V. cholerae species (Domman et al 2017 Americas paper) showing the lineages ( explain trees)
-brief mention that epidemic/pandemic cholera is caused by 7PET and spread by human travel & poor sanitation, 

The *V. cholerae* Genome and Whole-Genome Sequencing (WGS) of *V. cholerae*
---------------------------------------------------------------------------

Just as for us humans, the genetic material of *V. cholerae* consists of DNA.
The DNA of *V. cholerae* contains all the genetic instructions specifying the development of a *V. cholerae* cell. 

You may be already familiar with the structure of DNA, a famous molecule with a double helix structure. DNA molecules consist of two chains (also known as 'strands') of
smaller molecules called 'nucleotides'. Each nucleotide consists of three parts: a sugar called deoxyribose,
a phosphate group, and one of four 'bases'. The bases are thymine (abbreviated as 'T'), adenine ('A'), guanine ('G')
and cytosine ('C'). 

.. figure:: DNA_structure.jpg
  :width: 300

  A diagram of the structure of DNA, showing the sugar phosphate backbone and base-pairs. The sugars and phosphates form the backbone of the double helix. Image credit: National Human Genome Research Institute, Public domain, via Wikimedia Commons.

The bases in the two strands of a DNA double helix are 'complementary' to each other: that is, T pairs with
A and G pairs with C. Thus, if one strand has the sequence of bases TACGA, the other strand must have the sequence
of bases ATGCT. For convenience, one strand in a DNA double helix is called the 'forward' or '+' ('plus') strand, and the 
other strand the 'reverse' or '-' ('minus') strand.

The 'genome' of *V. cholerae* is the name we give to the set of all DNA molecules in a *V. cholerae* cell.
In each cell, the *V. cholerae* genome is organised into two circular 'chromosomes', each consisting of a long circular DNA molecule. 
In total the two circular chromosomes contain about 4 million base-pairs (4 Mb) of base-pairs, where Chromosome 1 is about 3 Mb 
and Chromosome 2 about 1 million base-pairs (1 Mb; `Heidelberg et al 2000`_). 

.. _Heidelberg et al 2000: https://pubmed.ncbi.nlm.nih.gov/10952301/

.. figure:: Chromosomes.png
  :width: 800

  A diagram showing the two circular chromosomes of *V. cholerae*, for a typical 7PET isolate. The outside circles in blue represent protein-coding genes on the forward and reverse strand of the DNA. Between them, the two chromosomes of *V. cholerae* include almost 4000 genes in a typical 7PET isolate from the 7PET lineage. Other key features of the chromosomes are highlighted as green boxes, where the green box labelled '5' is the 'CTX region' containing the two genes *ctxA* and *ctxB*, which encode the A and B subunits of the cholera toxin, respectively. Image source: `Mutreja & Dougan 2020`_.

.. _Mutreja & Dougan 2020: https://pubmed.ncbi.nlm.nih.gov/31345641/

When we talk about 'sequencing the genome' of an organism, we mean figuring out the sequence of
bases on the strands of its DNA molecules. 
Later in this course we will talk about different methods for sequencing DNA.

The first time that a *V. cholerae* isolate's genome was sequenced was in the year 2000, for a *V. cholerae* 7PET strain called N16961 
that was originally isolated in Bangladesh (`Heidelberg et al 2000`_).
Since the year 2000, the genomes of thousands of other *V. cholerae* isolates have been sequenced. There are quite a lot of small differences
between different *V. cholerae* isolates' genomes (that is, there is genetic variation within *V. cholerae*), 
but the majority of *V. cholerae* isolates have two circular chromosomes that together contain about 4 Mb of DNA. 

Genes of *V. cholerae*
----------------------

Each of the two chromosomes of *V. cholerae* includes many hundred of genes. In 
Each gene comprises a segment of its DNA, typically hundreds or thousands
of base-pairs in length. A very common type of gene is a protein-coding gene, which is a stretch of
DNA which encodes (specifies the production of) particular proteins. For example,
*ctxA* and *ctxB* are two *V. cholerae* genes that encode the CtxA and CtxB proteins, the two
proteins that form cholera toxin. In total the two *V. cholerae* chromosomes contain almost 4000 protein-coding genes in a typical isolate
from the 7PET lineage (`Heidelberg et al 2000`_). 

The *ctxA* and *ctxB* genes are usually found close together in the *V. cholerae* genome, in a region known as the **CTX region**. 
Practically 100% of 7PET isolates contain the CTX region in their chromosome, and so produce cholera toxin, and therefore cause a human 
host to suffer acute watery diarrhoea. Only a small number of non-pandemic/non-epidemic lineages of *V. cholerae*, most
of which are relatively closely related to 7PET, have *ctxA* and *ctxB* genes (`Chun et al 2009`_).

.. _Heidelberg et al 2000: https://pubmed.ncbi.nlm.nih.gov/10952301/

.. _Chun et al 2009: https://pubmed.ncbi.nlm.nih.gov/19720995/

.. figure:: CtxRegion.png
  :width: 550

  A diagram showing the CTX region of Chromosome 1 of 7PET strain N16961. The blue arrows represent genes. The *ctxA* and *ctxB* genes encode the A and B subunits, respectively, of the cholera toxin protein complex. Image source: A. Coghlan, inspired by `Pant et al 2020`_.

.. _Pant et al 2020: https://pubmed.ncbi.nlm.nih.gov/31272871/

The Classical Biotype and the El Tor Biotype
--------------------------------------------

The 7th pandemic isolates are described as having the **'El Tor biotype'**, to distinguish them from earlier form of *V. cholerae* that caused 
the 6th pandemic, which were described to as having the **'Classical biotype'**. These biotypes are based on a particular set of laboratory tests (about four
or five laboratory tests, depending on the version of the protocol), which test things like susceptibility to infection by particular bacteriophages, 
susceptibility to a particular antibiotic, presence of a particular biochemical pathway, and effect on red blood cells (`CDC 1994`_).  
The current pandemic lineage of *V. cholerae* is called '7PET' because it shows the El Tor biotype: '7PET'
stands for '7th pandemic El Tor'. 

.. _CDC 1994: https://stacks.cdc.gov/view/cdc/52473

.. figure:: LabTests.png
  :width: 250

  A set of about four or five laboratory tests are used to determine the biotype of *V. cholerae*, which is a laboratory phenotype that can be used to predict whether isolates of the *V. cholerae* causing a particular outbreak are from the 7th pandemic (i.e. belong to the 7PET lineage). Image credit: DynV at openclipart.org.

For many decades these laboratory tests have been used to find out whether the *V. cholerae*
causing an outbreak have the El Tor biotype (`CDC 1994`_), so to predict whether they are likely to be pandemic-causing/epidemic-causing *V. cholerae* (7PET). 
However, although the lineage causing the 7th pandemic, 7PET, shows the El Tor biotype in laboratory tests, some other
lineages of *V. cholerae* also show the El Tor biotype in laboratory tests (see Fig. 1. in `Chun et al 2009`_). 
As we will discuss later in this course, whole genome sequencing tells us far more accurately whether isolates from an 
outbreak are pandemic-causing/epidemic-causing *V. cholerae* (i.e. are 7PET). 

.. _Chun et al 2009: https://pubmed.ncbi.nlm.nih.gov/19720995/

Interestingly, we now know that, at the genetic level, the biotype phenotype is determined by several 
different genes (`Dorman 2020`_). The difference in biotype between isolates from the 6th and 7th pandemic 
reflects that there are genetic differences in these biotype-determining genes between the *V. cholerae* that caused these two pandemics (`Dorman 2020`_). 



Genetic Variation in *V. cholerae*
----------------------------------

-Discuss possible presence of the cholera toxin genes that are found as part of the cholera prophage region, possible presence of plasmids containing AMR genes.
N16961 is the “reference strain”
Rare V. cholerae non-O1/O139 have the two chromosomes fused into one (Johnson et al 2015) 
Rare V. cholerae O1 El Tor have an additional 0.9-Mb replicon; genome size 5.0 Mb (Okada et al 2015)

7PET
----

-its emergence in the 1960s, its very high virulence & very high transmissibility; its causing the 7th pandemic and causing all large cholera outbreaks & epidemics since the 1960s; brief mention of the need for a rapid public health response (WASH, treatment centres, contact tracing, public information campaign, vaccination, etc.) as soon as possible at the start of a 7PET outbreak, to contain/reduce/stop it;
-brief mention of serogroup as a likely virulence factor, and that 7PET usually has O1 (mostly) or O139 serogroup (sometimes), but that some other lineages are also O1 or O139;
-a brief history of the global spread of the current pandemic lineage (7PET), including mention of the Bay of Bengal being a region where cholera is endemic and from where 7PET has spread around the world, and brief history of introductions of 7PET into Africa; brief mention of countries worldwide where 7PET is now endemic;
-brief mention of how 7PET has split into sublineages, and brief mention that new sublineages have arisen over time in the Bay of Bengal region and spread around the world; brief mention of names of sublineages previously or currently found in Africa;
-the recently revised GTFCC definitions of cholera cases and ‘cholera-free’ country statuses;
-brief mention that 7PET can be controlled by vaccination; that vaccines have high efficiency against 7PET but are in short supply and that genomic information has great potential to help us decide who to vaccine (e.g. populations exposed to 7PET, especially those exposed to multi-drug resistant 7PET sublineages)

Brief Summary of Take-home Messages
-----------------------------------

The key take-home messages of this chapter are:

* Cholera is characterised by acute watery diarrhoea
* Cholera is caused by ingestion of *Vibrio cholerae*
* The acute water diarrhoea is triggered by cholera toxin produced by *V. cholerae*
* Cholera toxin is the key virulence factor of *V. cholerae* 
* *V. cholerae* is distributed globally, and is a very diverse species with many different lineages 
* Most lineages of *V. cholerae* do not cause pandemic/epidemic cholera
* At present there is only one lineage that causes pandemic/epidemic cholera: 7PET 
* **7PET is the only high-risk lineage of Vibrio cholerae at present: it is extremely infectious and virulent**
* Biotype is a laboratory phenotype used to predict whether *V. cholerae* isolates are epidemic-causing/pandemic-causing (i.e. 7PET)
* Biotype is not a perfect predictor of 7PET, because a small % of non-pandemic/non-epidemic lineages have the El Tor biotype
* The genome of a 7PET isolate has 4 Mb split over two chromosomes, and contains about 4000 genes
* Practically all 7PET isolates have genes for cholera toxin (*ctxAB* genes)
* Presence of *ctxAB* is not a perfect predictor of 7PET, because a small % of non-pandemic/non-epidemic lineages have *ctxAB* 
* **Whole genome sequencing is the most accurate way to determine whether an outbreak is 7PET (and so pandemic-causing/epidemic-causing)**
* **A 7PET outbreak requires a rapid and large public health response**

Suggested Further Reading
-------------------------

Suggested further reading to go with this chapter is the nice review by `Mutreja & Dougan 2020`_ on "Molecular epidemiology and intercontinental spread of cholera". 

.. _Mutreja & Dougan 2020: https://pubmed.ncbi.nlm.nih.gov/31345641/

Contact
-------

I will be grateful if you will send me (Avril Coghlan) corrections or suggestions for improvements to my email address alc@sanger.ac.uk

Acknowledgements
----------------

Contributors to this course: Avril Coghlan, Nick Thomson. 


